pkg/sql/gcjob_test/gc_job_test.go

```diff
@@ -317,10 +317,30 @@ SELECT job_id
   FROM [SHOW JOBS]
  WHERE job_type = 'SCHEMA CHANGE GC' AND description LIKE '%foo%';`,
 	).Scan(&jobID)
-	tdb.CheckQueryResultsRetry(t,
-		"SELECT running_status FROM crdb_internal.jobs WHERE job_id = "+jobID,
-		[][]string{{string(sql.RunningStatusWaitingForMVCCGC)}},
-	)
+
+	const expectedRunningStatus = string(sql.RunningStatusWaitingForMVCCGC)
+	testutils.SucceedsSoon(t, func() error {
+		var runningStatus, lastRun, nextRun, numRuns, jobErr string
+		tdb.QueryRow(t, fmt.Sprintf(`
+SELECT running_status, error, last_run, next_run, num_runs
```

Comment:
i think it would be useful to query `status` as well

Reply:
Added!